api/src/test/java/org/pmiops/workbench/api/util/SQLGeneratorTest.java

```diff
@@ -122,64 +135,58 @@ public void findParametersWithEmptyDomainIds() throws Exception {
 
     @Test
     public void getSubQuery() throws Exception {
+        String TABLE_PREFIX = workbenchConfig.bigquery.projectId + "." + workbenchConfig.bigquery.dataSetId;
         Map<String, String> expectedByKey = new HashMap<String, String>();
         expectedByKey.put("Condition",
             "SELECT DISTINCT PERSON_ID " +
```

Comment:
Nit: make all table names and column names lower case? 

Making everything all caps makes it hard to parse.

Reply:
https://github.com/all-of-us/workbench/pull/159#discussion_r141481302

Column names are actually *not* case sensitive (unlike table names), so we _could_ make them all lowercase, but the table names would still need addressing in the db itself.